master/internal/api_experiment.go

```diff
@@ -2195,7 +2197,9 @@ func sortExperiments(sortString *string, experimentQuery *bun.SelectQuery) error
 			WHERE t.experiment_id = e.id
 			ORDER BY searcher_metric_value_signed ASC
 			LIMIT 1
-		 ) `,
+		) `,
+		"externalExperimentId": "e.external_experiment_id",
+		"externalTrialId":      "t.external_trial_id",
```

Comment:
This should be `trials.external_trial_id`

Reply:
fixed